public/jsx/components/card-form.jsx

```diff
@@ -54,6 +62,10 @@ module.exports = React.createClass({
     };
   },
 
+  contextTypes: {
+    router: React.PropTypes.func,
+  },
```

Comment:
This along with the mixins seem like something that would need to be in all of our components. If that's the case, could we make a base component to inherit from?